src/sagemaker/analytics.py

```diff
@@ -450,6 +452,11 @@ def __init__(
             sagemaker_session (sagemaker.session.Session): Session object which manages interactions
                 with Amazon SageMaker APIs and any other AWS services needed. If not specified,
                 one is created using the default AWS configuration chain.
+            input_artifact_names(dict optional):The input artifacts for the experiment. Examples of
+             input artifacts are datasets, algorithms, hyperparameters, source code, and instance
```

Comment:
Could you correct the indentations in the docstring?

Reply:
fixed